R/sendQuery.R

```diff
@@ -135,15 +135,15 @@ sendQuery(db ~ DBIConnection, query ~ SingleQuery, ...) %m% {
 
 #' @export
 #' @rdname sendQuery
-sendQuery(db ~ MySQLConnection, query ~ SingleQuery, ..., encoding = "utf8") %m% {
+sendQuery(db ~ MySQLConnection, query ~ SingleQuery, ..., encoding = "utf8mb4") %m% {
```

Comment:
@jonathan-inwt ist das tatsächlich rückwärts kompatibel?

Reply:
@wahani Soweit ich das verstehe, ja.

https://dev.mysql.com/doc/refman/5.5/en/charset-unicode-utf8mb4.html
> For a BMP character, utf8mb4 and utf8mb3 have identical storage characteristics: same code values, same encoding, same length.

https://mariadb.com/kb/en/unicode/
> utf8mb4: Same as utf8, but stores supplementary characters in four bytes.


https://dev.mysql.com/doc/refman/5.5/en/charset-unicode-utf8mb4.html
> The character set named utf8[/utf8mb3] uses a maximum of three bytes per character and contains only BMP characters. As of MySQL 5.5.3, the utf8mb4 character set uses a maximum of four bytes per character supports supplemental characters:
> 
> For a BMP character, utf8[/utf8mb3] and utf8mb4 have identical storage characteristics: same code values, same encoding, same length.
> 
> For a supplementary character, utf8[/utf8mb3] cannot store the character at all, while utf8mb4 requires four bytes to store it. Since utf8[/utf8mb3] cannot store the character at all, you do not have any supplementary characters in utf8[/utf8mb3] columns and you need not worry about converting characters or losing data when upgrading utf8[/utf8mb3] data from older versions of MySQL.